btc-dw-bridge/src/main/kotlin/com/d3/btc/dwbridge/config/BtcDWBridgeConfig.kt

```diff
@@ -25,5 +26,16 @@ interface BtcDWBridgeConfig {
     // Port for web services
     val webPort: Int
 
-    val dnsSeedAddress: String?
+    // Bitcoin DNS seeds. Seeds are separated with comma (',') symbol
+    val dnsSeedAddresses: String?
+
+    companion object {
+        fun extractSeeds(btcDWBridgeConfig: BtcDWBridgeConfig): List<String> {
+            return if (btcDWBridgeConfig.dnsSeedAddresses == null) {
```

Comment:
how about making `extractCommaSeparatedList` with `String?` argument and check `isNullOrEmpty` there?

Reply:
Cool